ansible/all-in-one/roles/sockets/files/http.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
```

Comment:
How can we be sure that this is started after `nginx` inside podman? If I understand correctly, currently we may getting `502 Bad Request` if proxy is started but nginx is not.

Reply:
Oh.. yeah. Hm.